static/app/components/superuserAccessForm.tsx

```diff
@@ -160,8 +166,15 @@ class SuperuserAccessForm extends Component<Props, State> {
       this.handleLogout();
       return null;
     }
+
     return (
       <ThemeAndStyleProvider>
+        {this.props.hasStaff ? // Skip access form if using new new staff
```

Comment:
what's `new new staff`? 

Thought - Can we just refer to these as `staff` vs. `elevated` permissions or something?